rds.go

```diff
@@ -22,8 +22,14 @@ const (
 	InstanceNotGone                           // 4
 )
 
+type DatabaseStatus struct {
+	State        string       `json:"state"`
+	Description  string       `json:"description"`
+}
+
```

Comment:
For maintainability, you can create a type like
`type InstanceState string`
then create a const of all your possible states

created a playground link to show the safety you get: https://play.golang.org/p/V6i-RagAX-